app/models/healthcheck.rb

```diff
@@ -38,26 +38,24 @@ def test_gitis
   end
 
   def test_dfe_signin_api
-    return true unless Schools::DFESignInAPI::Organisations.enabled?
-
-    Schools::DFESignInAPI::Organisations.new(SecureRandom.uuid).uuids
+    !Schools::DFESignInAPI::Organisations.new(SecureRandom.uuid).uuids.nil?
```

Comment:
```suggestion
    Schools::DFESignInAPI::Organisations.new(SecureRandom.uuid).uuids.any?
```

Reply:
We can't use `#any?` or `#present?` because `Schools::DFESignInAPI::Organisations.new(SecureRandom.uuid).uuids` returns an empty hash, used as a boolean value in line 52.

Now, `{}` acts as a `true` in some cases when doing logical calculations:

```
pry(main)> {} && true
=> true
[5] pry(main)> true && {}
=> {}
[6] pry(main)> {} || false
=> {}
[7] pry(main)> false || {}
=> {}
```

So what I was trying to do is cast `{}` to a boolean value but the [linter changed](https://rubystyle.guide#no-bang-bang) it to this `!foo.nil?`, which evaluates to true when `foo = {}` and false when nil.

Don't like it either, so let me use ActiveModel::Type::Boolean.new.cast() to make the intention clearer.